test/metabase/search/fulltext/scoring_test.clj

```diff
@@ -79,9 +79,40 @@
               ["card" 1 "card ancient"]]
              (search :recency "card"))))))
 
+(deftest ^:parallel view-count-test
+  (testing "the more view count the better"
+    (with-index-contents
+      [{:model "card" :id 1 :name "card famous"     :view_count 100}
+       {:model "card" :id 2 :name "card popular"    :view_count 50}
+       {:model "card" :id 3 :name "card well known" :view_count 10}]
+      (is (= [["card" 1 "card famous"]
```

Comment:
We max out the score at the p99, it would be good to test this edge case by having something exceed it.

Reply:
hmm, how do we do that?

Reply:
Fill the db with junk, or use `with-dynamic-redefs` on `view-count-percentiles`.

It would be good to have some test coverage of the query, if you go the latter route.